controller/Lists.php

```diff
@@ -64,38 +63,44 @@
 class Lists extends tao_actions_CommonModule
 {
     use OntologyAwareTrait;
+    use HttpJsonResponseTrait;
 
     private const REMOTE_LIST_PREVIEW_LIMIT = 20;
 
     /** @var bool */
     private $isListsDependencyEnabled;
 
     /**
-     * Show the list of users
+     * This REST endpoint:
+     * - Returns the page with the lists for GET requests
+     * - Creates a new list and returns its name and URI for POST requests
+     *
      * @return void
      */
     public function index()
     {
-        tao_helpers_Scriptloader::addCssFile(Template::css('lists.css', 'tao'));
-        $this->defaultData();
-
-        $myAdderFormContainer = new tao_actions_form_List();
-        $myForm = $myAdderFormContainer->getForm();
-
-        if ($myForm->isSubmited()) {
-            if ($myForm->isValid()) {
-                $values = $myForm->getValues();
-                $newList = $this->getListService()->createList($values['label']);
-                $i = 0;
-                while ($i < $values['size']) {
-                    $this->getListService()->createListElement($newList, __('element') . ' ' . ($i + 1));
-                    $i++;
-                }
+        if ($this->getPsrRequest()->getMethod() == 'POST') {
```

Comment:
Please, use strict comparison

Reply:
Changed in 6689211726e94157e9111ee267bdf4e5a4a49f79, thanks :+1: